### DISCUSSION TITLE
docker  curl

### DISCUSSION THREAD

**Discussion by wangqingpen** [edgexfoundry/community#252] [Q&A]
When I access 59881 the native cannot be accessed, it can be accessed externally
Native：
![1f5a52ba699315cb8cb355e6a4ce93b](https://github.com/edgexfoundry/community/assets/72652818/27aa2d8d-669b-4191-9c65-20737c684ec0)
outside：
![c9cd6aa3443760719e3af516169b9e1](https://github.com/edgexfoundry/community/assets/72652818/a4869ce2-9fc1-483a-baed-0eb29fac9526)

**Comment by lenny-goodell** [comment]
This is because the IP address you are using is for the host system, which is unknown from within the docker container.

**Reply by wangqingpen** [reply]
Where can I troubleshoot the cause of this problem?

**Reply by lenny-goodell** [reply]
The cause is the IP doesn't exist in the container. You can use `127.0.0.1` or the docker network address of the container.

**Reply by jiekechoo** [reply]
@wangqingpen Try to use `0.0.0.0` instead of the host ip.

**Reply by wangqingpen** [reply]
@jiekechoo  I tried changing to 0.0.0.0 and it's still the same error, is there any other way

**Comment by wangqingpen** [comment]
> @wangqingpen Try to use `0.0.0.0` instead of the host ip.

Have you used host, can you provide an example of docker.yml?

**Reply by jiekechoo** [reply]
Just replace `host_ip` value to `0.0.0.0` in your docker-compose.yml.
```
    ports:
      - mode: ingress
        host_ip: 0.0.0.0
```

**Comment by cloudxxx8** [comment]
@wangqingpen you may provide more your steps and context.  it's basically a network issue.
